Tags specs (#268)
PrestaShop/prestashop-specs: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md

> ---

Reviewer: Can you rename content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md by content/1.7/back-office/shop-parameters/search/tags/add-edit-tag.md as you add and edit only one tag, the S should be removed

> title: Add / Edit Store

Reviewer: ```suggestion
title: Add / Edit tag
```

weight: 1
---

> # Add / Edit tags 

Reviewer: ```suggestion
# Add / Edit tag
```

> While adding or editing tags, the following fields/drop downs/parameters are displayed:

Reviewer: ```suggestion
While adding or editing a tag, the following fields/drop downs/parameters are displayed:
```

1) Name 

> This field is for the name of the tag, the field is mandatory. When trying to save without any name input, shows the message: "The name field is required."

Reviewer: ```suggestion
This field allows to fill the name of the tag and is mandatory. If the save button is pressed and if this field is empty, the following error message is displayed: "The name field is required."
```

2) Language

> This is a drop-down menu for the language of the tag. The default country (selected in International > Localization > Configuration) is selected by default.

Reviewer: ```suggestion
This drop-down menu allows selecting the language of the tag. The default country (selected in International > Localization > Configuration) is selected by default.
```

> 

Reviewer: The countries listed in the drop-down are the countries enabled in International > Localization > Languages

3) Products

> This section has 2 fields, the one on the left indicates all the products that are added to the shop, and has a button below the list named: "Add". When selecting a product, and pressing "add" the selected product moves to the right box, which lists the products that are listed with the selected tag.

Reviewer: ```suggestion
This section allows to select products associated with the tag and has 2 parts:
- the one on the left lists all the products of the shop
- the one on the right lists all the products selected for the tag.
 When selecting a product, and clicking on "add", the selected product moves to the right box.
The user can select several products and add them to this tag at once.
```

> From the field box o the right, you can also select the products, and press "remove" which is below the field, then the product is moved back to the left field box.

Reviewer: ```suggestion
When selecting a product in the list on the right, and clicking on "remove", the selected product is moved back to the left box.
```
